### PR TITLE
Make it clear that documentation is out dated

### PR DESCRIPTION
**Pull request by vaceletm**
Take the opportunity to remove old/outdated pieces.

### CODE REVIEW

File: languages/fr/user-guide/cvs.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
    :depth: 3
 ..
 
+.. attention::
```
Comment (nterray) [inline languages/fr/user-guide/cvs.rst:5]: Could you please put it above the toc like for index?

File: languages/fr/user-guide/git.rst [hunk 2]
```diff
@@ -2,6 +2,14 @@
    :depth: 3
 ..
 
+.. attention::
```
Comment (nterray) [inline languages/fr/user-guide/git.rst:5]: Could you please put it above the toc like for index?

File: languages/fr/user-guide/doc.rst [hunk 3]
```diff
@@ -2,6 +2,12 @@
    :depth: 3
 ..
 
+.. attention::
```
Comment (nterray) [inline languages/fr/user-guide/doc.rst:5]: Could you please put it above the toc like for index?`

**Review by nterray — CHANGES_REQUESTED** [review]

File: languages/fr/index.rst [hunk 4]
```diff
@@ -1,6 +1,12 @@
 Bienvenue sur la documentation de Tuleap !
 ==========================================
 
+.. attention::
+
+   La version française de la documentation est n'est pas régulièrement mise à jour. 
+   
+   Il est recommandé de se réfèrer à la version `anglaise </doc/en/>`_.
```
[the remark this answers is not in the document]
Comment (vaceletm) [reply languages/fr/index.rst:8]: It's the default configuration shipped with Tuleap nginx configuration. I doubt someone changed that.

**Review by LeSuisse — APPROVED** [review]

**Pull request merged**